qiskit_ibm_provider/ibm_backend.py

```diff
@@ -482,11 +485,23 @@ def run(
             # Transpiling in circuit-runner is deprecated.
             run_config_dict["skip_transpilation"] = True
 
-        if isinstance(circuits, (QuantumCircuit, Schedule)):
+        if isinstance(circuits, Schedule):
+            raise IBMBackendValueError(
+                "Class 'Schedule' is no longer supported as "
+                "an input circuit. See 'run' method documentation"
```

Comment:
Why not mention the pulse gate with link right in the message? So users don't need to hunt down the docs